process/jira.md

```diff
@@ -0,0 +1,54 @@
+# Jira Hygiene
+
+As almost anything even Jira requires regular maintenance to address:
+
+-   Incomplete/unclear tickets
+-   Stale/outdated tickets
+-   Forgotten/unpopular tickets
+-   Usefulness of Jira content as a whole
+
+Why? Even though it might not look like it at first glance, Jira is a tool to *communicate with* people in an open, transparent and remote-friendly manner and facilitate *reaching our goals*. This text tries to apply our [PLOW](https://culture.mayadata.io/plow/plow.html) to working with Jira. To keep various Jira projects open and welcoming. To allow us to cooperate effectively and win - personally and professionally and deliver the value that is useful to the company and open-source community in a repeatable and dependable manner.
+
+## Two loops
+
+For Jira to be an effective communication tool, information must flow freely while being taken care of at every stage.
+
+### Loop one \-- Initial Triage
+
+Person responsible for maintenance should regularly \-- **at least daily** \-- check new tickets flowing into the project (status *open*). In case of projects that can receive time sensitive tasks (e.g. operations/SRE, customer support) it is recommended to set up a notification channel for new tickets (if you like email - see [PLOW-22](https://mayadata.atlassian.net/browse/PLOW-22);
+there's a possibility to send new tickets to slack channel but consider that shared responsibility often doesn't work and unnecessary interrupts and notifications are especially harmful to deep complex work we do here, so it is recommended to point this integration to a special channel for this purpose. Especially avoid channels with more than few involved members).
+
+It is required to take care of every new ticket, it does not mean that every ticket will be worked on when received. Probably incomplete list of what to do with new tasks:
+
+-   Ticket is **clear, understandable** and **actionable**, contains **why**, **what** and any particularities required. Also has all required metadata. It is aligned with team priorities, [OKRs](https://culture.mayadata.io/process/okr.html) and company's strategy. Such tickets should be *accepted for prioritization*. In case of urgent tickets these must skip prioritization and must be taken care of immediately with clearly agreed ownership.
```

Comment:
```suggestion
-   Ticket is **clear**, **understandable** and **actionable**, contains **why**, **what** and any particularities required. Also has all required metadata. It is aligned with team priorities, [OKRs](https://culture.mayadata.io/process/okr.html) and company's strategy. Such tickets should be *accepted for prioritization*. In case of urgent tickets these must skip prioritization and must be taken care of immediately with clearly agreed ownership.
```